src/styles/utils/variables.scss

```
@@ -0,0 +1,5 @@
+$color-primary: #00acdc;
+$color-secondary: #060b35;
+$color-text: #616070;
+$font-primary: Roboto, sans-serif;
+$nav-link-color: #060b35;
```

Why did you create two variables with the same values?
```suggestion
$color-secondary: #060b35;
$color-text: #616070;
$font-primary: Roboto, sans-serif;
```